webtau-graphql/src/main/java/org/testingisdocumenting/webtau/graphql/GraphQLCoverage.java

```diff
@@ -38,16 +38,34 @@ public void recordQuery(HttpValidationResult validationResult) {
         }
 
         Set<GraphQLQuery> graphQLQueries = schema.findQueries(validationResult.getRequestBody());
-        graphQLQueries.forEach(query -> coveredQueries.add(query, validationResult.getId(), validationResult.getElapsedTime()));
+        graphQLQueries.forEach(query -> coveredQueries.
+            add(query, validationResult.getId(), validationResult.getElapsedTime(), isErrorResult(validationResult)));
     }
 
-    Stream<GraphQLQuery> nonCoveredQueries() {
-        return schema.getSchemaDeclaredQueries().filter(o -> !coveredQueries.contains(o));
+    private boolean isErrorResult(HttpValidationResult validationResult) {
+        return JsonUtils.convertToTree(JsonUtils.deserialize(validationResult.getResponse().getTextContent()))
```

Comment:
Could this throw? If it can, we probably should catch, log and move on without throwing.

Reply:
Very interesting point.
There seems to be no standard logging in reporters or elsewhere. I agree that we should be able to move on if something goes wrong here. I'd say if an error cannot be detected, it's not an `errorResult`, but how visible is a stacktrace or other error log here?

Reply:
There is some precedent for ignoring these sorts of issues: https://github.com/testingisdocumenting/webtau/blob/master/webtau-graphql/src/main/java/org/testingisdocumenting/webtau/graphql/GraphQLSchema.java#L68 but you're right, it's generally silent which is not ideal.